app/graphql/mutations/create_vimeo_video.rb

```diff
@@ -0,0 +1,22 @@
+module Mutations
+  class CreateVimeoVideo < GraphQL::Schema::Mutation
+    argument :size, Integer, required: true
+    argument :target_id, ID, required: true
```

Comment:
This doesn't seem to be used anywhere, and I'm pretty sure it isn't required to let a user upload a video to Vimeo.